src/main/kotlin/com/bridgecrew/ui/buttons/SuppressionButton.kt

```diff
@@ -0,0 +1,18 @@
+package com.bridgecrew.ui.buttons
+
+import java.awt.event.ActionEvent
+import java.awt.event.ActionListener
+import javax.swing.JOptionPane
+
+const val suppressionButtonText = "Suppression"
```

Comment:
shouldn't show `suppress` (like call to action)?

Reply:
you're right